### PR TITLE
Add vendor tarballs to releases

### PR DESCRIPTION
**Pull request by Username404-59**
[~~not tested yet~~](https://github.com/Username404-59/vinegar/releases/tag/v1.4.3)

### CODE REVIEW

**Review by apprehensions — COMMENTED** [review]
lbtm

File: .github/workflows/vendor.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: 'Make the vendor directory'
+        run: go mod vendor
+      - name: 'Package the source directory'
+        run: |
+          export RELEASE="vinegar-${{ github.ref_name }}"
+          cd ..
+          cp -r ./vinegar ./$RELEASE
+          XZ_OPT=-9 tar -Jcf $RELEASE.tar.xz \
+            --exclude .git \
+            ./$RELEASE
```
Comment (apprehensions) [inline .github/workflows/vendor.yml:26]:
```suggestion
          RELEASE="vinegar-${{ github.ref_name }}"
          
          cd ..
          cp -r vinegar $RELEASE
          
          XZ_OPT=-9 tar -Jcf $RELEASE.tar.xz \
            --exclude .git \
            $RELEASE
            
         mv $RELEASE vinegar
```

**Comment by Thelolguy1** [comment]
This merge will require effort to implement on the Flatpak end; the amount of work to transition to tarballs is unknown at the moment.

**Comment by Username404-59** [comment]
> This merge will require effort to implement on the Flatpak end; the amount of work to transition to tarballs is unknown at the moment.

I believe it should be as simple as changing the sources type to `archive`, changing the url, and finally unpacking it if flatpa k doesn't do it

**Comment by Thelolguy1** [comment]
good advice, will try it when this is merged

**Review by apprehensions — APPROVED** [review]

**Pull request merged**